code/modules/reagents/chemistry/colors.dm

```diff
@@ -16,14 +16,16 @@ GLOBAL_LIST_INIT(random_color_list, list("#00aedb","#a200ff","#f47835","#d41243"
 	for(var/datum/reagent/reagent in reagent_list)
 		if(reagent.id == "blood" && reagent.data && reagent.data["blood_color"])
 			reagent_color = reagent.data["blood_color"]
+		else if(reagent.id == "slimejelly" && reagent.data && reagent.data["colour"])
+			reagent_color = reagent.data["colour"]
 		else
 			reagent_color = reagent.color
 
 		vol_temp = reagent.volume
 		vol_counter += vol_temp
 
-		if(isnull(color))
-			color = reagent.color
+		if(isnull(color)) // `color` will always be null so I'm not sure what these `else`s are for.
```

Comment:
Regarding the comment: 
We're inside a for loop, so the `isnull` is essentially saying "do this thing on the first iteration only".

Reply:
Ah, alright that makes sense.